securedrop/config.py.example

```diff
@@ -50,18 +50,24 @@ env = os.environ.get('SECUREDROP_ENV') or 'prod'
 if env == 'prod':
     # This is recommended for performance, and also resolves #369
     FlaskConfig.USE_X_SENDFILE = True
+    # How long a session is valid before it expires and logs a user out
+    SESSION_EXPIRATION_MINUTES = 30
 elif env == 'dev':
     # Enable Flask's debugger for development
     FlaskConfig.DEBUG = True
     # Use MAX_CONTENT_LENGTH to mimic the behavior of Apache's LimitRequestBody
     # in the development environment. See #1714.
     FlaskConfig.MAX_CONTENT_LENGTH = 524288000
+    # How long a session is valid before it expires and logs a user out
+    SESSION_EXPIRATION_MINUTES = 30
 elif env == 'test':
     FlaskConfig.TESTING = True
     # Disable CSRF checks to make writing tests easier
     FlaskConfig.WTF_CSRF_ENABLED = False
     # TODO use a unique temporary directory for each test so we can parallelize them
     SECUREDROP_DATA_ROOT = '/tmp/securedrop'
+    # Use a short session expiration for functional tests
+    SESSION_EXPIRATION_MINUTES = 0.5
```

Comment:
This is likely to cause problems for other functional tests on slow machines if a given functional test runs for more than 30 seconds for any reason. Would it be possible to parameterize the functional test test_session_timeout_message with the SESSION_EXPIRATION_MINUTES so that functional_test.py::setup launches the app with this expiration ?

Reply:
:+1: I'll take a look at this approach